src/main/java/frc/robot/RobotContainer.java

```diff
@@ -173,6 +196,17 @@ private void configureBindings() {
             new InstantCommand(() -> drive.setFieldOrientedDrive(!drive.isFieldOrientedDrive())));
 
     controller.back().onTrue(new InstantCommand(() -> drive.resetOdometry()));
+
```

Comment:
These should be commented out so that we don't affect the actual controls in baseline.